src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/4.sql

```diff
@@ -407,7 +407,7 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_ExtendedQueryTag_TagPath ON dbo.ExtendedQuer
 **************************************************************/
 CREATE TABLE dbo.ExtendedQueryTagOperation (
     TagKey                  INT                  NOT NULL, --PK
-    OperationId             VARCHAR(32)          NOT NULL -- TODO: Use uniqueidentifier
+    OperationId             uniqueidentifier     NOT NULL
```

Comment:
Should we also change {study-series-sop}instanceUids to be uniqueidentifier instead of varchar?

Reply:
While these are "Unique Identifiers" (UI) according to the DICOM spec, their representation cannot be reconciled with the .NET `Guid` type. While the DICOM UI uses at most 64 bytes, the .NET GUID (or UUID) uses 16 bytes (128 bits).